stdlib/Distributed/src/cluster.jl

```diff
@@ -1,7 +1,45 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    ClusterManager
+
+Supertype for cluster managers, which control how nodes are added to a cluster, driven within it, and released from it.
```

Comment:
`driven within it` doesn't read smooth to be. Maybe:

`Supertype for cluster manager, which controls workers processes as a cluster. Cluster managers implement how workers can be added, removed and communicated with.`